backend/app/core/utils/querying_utils.py

```diff
@@ -96,6 +98,32 @@ def logout(session_id: str) -> None:
 			session.delete(user_session)
 			session.commit()
 
+	@staticmethod
+	def get_user_info(session_id: str) -> UserOut:
+		with Session(engine) as session:
+			session_data = session.exec(
+				select(SessionModel).where(SessionModel.id == session_id)
+			).first()
+			if not session_data:
+				raise UserNotFoundError
+
+			user = session.exec(select(User).where(User.id == session_data.user_id)).first()
+			if not user:
+				raise UserNotFoundError
+
+			auth = session.exec(select(Auth).where(Auth.id == user.auth_id)).first()
+			if not auth:
+				raise UserNotFoundError
+
+			user_out = UserOut(
+				first_name=user.name,
+				last_name=user.last_name,
+				hashed_email=ModelUtils.hash_password(auth.email),
```

Comment:
this will result in different value every time. This is undesired, as we would like to always display the same avatar for a distinct user

Reply:
I would suggest using a deterministic hash like sha256, as email is not a critical security component